src/dxtbx/nexus/__init__.py

```diff
@@ -14,6 +14,8 @@
 from dxtbx import flumpy
 from dxtbx.nexus.nxmx import units
 
+from itertools import groupby
```

Comment:
```suggestion
import itertools
```
A matter of personal preference, but I find it helps to make it clearer what `groupby` function you are calling later. Given that you only use `groupby` once in this file, I'm not sure there's much to be gained by `from module import thing` over `import module`.